Update TMC and developer list

Mateasek opened the pull request:
I completely forgot this action assigned to me last spring. I would like to ask @jacklovell, @munechika-koyo and @skuba31 to check their affiliations, names and suggest any possible improvements or additions to this update.

Mateasek committed:
Update TMC and developer list

jacklovell commented:
Looks good, thanks! Can you update my affiliation to "Oak Ridge National Laboratory" instead of just "Oak Ridge"? I'd also suggest "former" in place of "honorary" for TMC members who are no longer active.

Mateasek committed:
Use the full and correct affiliation

Use Former instead of Honorary

Mateasek commented:
Thanks for the suggestions @jacklovell , I changed Honorary to Former. Also, I corrected your affiliation and used the full affiliation of IPP Prague. You made me realize we should use the correct full ones.

jacklovell commented:
Thanks, looks good to me. I'll wait for @munechika-koyo and @skuba31 to confirm they're happy and then we can merge.

munechika-koyo commented on docs/source/welcome.rst:
```diff
@@ -20,10 +20,11 @@ The following authors have contributed to the project:
+* Jack Lovell (Oak Ridge National Laboratory, USA)
+* Vlad Neverov
+* Matej Tomes (Institute of Plasma Physics of the Czech Academy of Sciences, Czechia)
+* Koyo Munechika (ITER Organisation)
```
```suggestion
* Koyo Munechika (ITER Organization)
```
Just a small change

munechika-koyo commented on README.md:
```diff
@@ -85,12 +85,19 @@ of code / physics algorithm quality standards.
 
+- Jack Lovell (Oak Ridge National Laboratory, USA)
+- Matej Tomes (Institute of Plasma Physics of the Czech Academy of Sciences, Czechia)
+- Koyo Munechika (ITER Organisation)
```
```suggestion
- Koyo Munechika (ITER Organization)
```

munechika-koyo commented:
@Mateasek Thank you for adding me!
I made a minor change to conform to the official IO spelling.

skuba31 commented:
Looks good. Thanks @Mateasek
